wiki/en/en-Installation-for-Windows.md

```diff
@@ -8,10 +8,10 @@ permalink: "/wiki/Installation-for-Windows"
 # Installation for Windows
 Make sure you read the [Getting Started](Getting-Started) page.
 1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have one (especially internal sound cards), install [this free ASIO driver (ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} before installing Jamulus.
```

Comment:
I wonder if we should say "If you don't have an external sound card", since it's more likely that those new to this idea will understand that if they don't have another box then it must be internal?

Reply:
Agree.

Reply:
There are some internal sound cards with ASIO drivers, I think (Realtek ASIO), but they are rare.